tests/__init__.py

```diff
@@ -1,11 +1,18 @@
 import sys
 from pathlib import Path
+from unittest import mock
 
 import pytest
+from pytest import MonkeyPatch
 
 sys.dangerzone_dev = True  # type: ignore[attr-defined]
 
+from dangerzone import util
 from dangerzone.document import SAFE_EXTENSION
+from dangerzone.gui import Application
+from dangerzone.gui.logic import DangerzoneGui
+from dangerzone.gui.updater import Updater
+from dangerzone.isolation_provider.dummy import Dummy
```

Comment:
We can do this, although this would mean that our Pytest fixtures would be in several places, which I think is not necessary since they are pretty few. But that's not deal-breaker and, since you've already done the work in splitting those, I'll follow suit.

Reply:
Should be fixed in e3b125d.

Reply:
Thanks!